jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaLineMarker.kt

```diff
@@ -45,7 +46,7 @@ class LambdaLineMarker : LineMarkerProviderDescriptor() {
         val handlerResolver = LambdaHandlerResolver.getInstance(runtimeGroup) ?: return null
         val handler = handlerResolver.determineHandler(element) ?: return null
 
-        return if (handlerResolver.shouldShowLineMarker(handler) || shouldShowLineMarker(element.project, handler, runtimeGroup)) {
```

Comment:
We do we remove this `handlerResolver.shouldShowLineMarker(handler)`?

Reply:
Added back